### DISCUSSION TITLE
Where is the data saved ?

### DISCUSSION THREAD

**Discussion by CubeBeveled** [internetarchive/heritrix3#585] [Q&A] [answered]
I could not find anything in the documentation or config files, the only thing i found is a website i wanted to crawl archived on `archive.today` (`archive.is`). I also checked `archive.org` and there weren't any snapshots.

Side question: Also is there an unofficial or official Discord server for the Internet Archive or heritrix ?

Note: I might be dumb

**Comment by CubeBeveled** [comment]
I figured out they get saved locally as warc files
But how do i upload them to archiving sites ?

**Comment by CubeBeveled** [comment]
I found [this](https://gist.github.com/Asparagirl/6206247?permalink_comment_id=4680900#gistcomment-4680900) that states that archive.org does not allow WARC uploads.
So this crawler is useless ?

**Comment by ato** [comment, marked as answer]
Heritrix saves data to WARC files in the `jobs/{jobname}/{timestamp}/warcs` subdirectory.

Most web archives only accept WARC files from trusted sources, not from the general public, as there isn't a way to guarantee the records are unaltered.
